neqo-transport/src/cc/cubic.rs

```diff
@@ -302,3 +480,35 @@ impl WindowAdjustment for Cubic {
         self.last_max_cwnd = last_max_cwnd;
     }
 }
+
+// UPDATE: Things to remember that didn't make it anywhere else:
+//
+// 1. Minimum congestion window:
+//
+// > Note that CUBIC MUST continue to reduce cwnd in response to congestion events
+// > detected by ECN-Echo ACKs until it reaches a value of 1 SMSS. If congestion events
+// > indicated by ECN-Echo ACKs persist, a sender with a cwnd of 1 SMSS MUST reduce its
+// > sending rate even further. This can be achieved by using a retransmission timer
+// > with exponential backoff, as described in [RFC3168].
+//
+// <https://datatracker.ietf.org/doc/html/rfc9438#section-4.6-7> VERSUS
+//
+// > QUIC therefore recommends that the minimum congestion window be two packets.
+//
+// <https://datatracker.ietf.org/doc/html/rfc9002#section-4.8>
+//
+// QUESTION: So this probably does not apply to CUBIC on QUIC and we never go below 2*MSS
+// (as is currently implemented)?
+//
+// 2. Timeout: <https://datatracker.ietf.org/doc/html/rfc9438#section-4.8>
+//
+// > QUIC does not collapse the congestion window when the PTO expires, since a single
+// > packet loss at the tail does not indicate persistent congestion.
+//
+// <https://datatracker.ietf.org/doc/html/rfc9002#section-4.7>
+//
+// QUESTION: Timeout (RTO/PTO) does not apply to us then?
+//
+// 3. Spurious fast retransmits: <https://datatracker.ietf.org/doc/html/rfc9438#section-4.9.2>
+//
+// QUESTION: Do we want to implement this? If yes then we may be able to do it in a follow up PR.
```

Comment:
I am in favor of doing this in a follow-up pull request. I suggest tracking it in a GitHub issue.